bin/Powheg/production/Run3/13p6TeV/DYToLL_NNPDF31_13p6TeV_MllBinned/Z_mumu_NNPDF31_13p6TeV_M_200_400.input

```diff
@@ -0,0 +1,40 @@
+! Z production parameter
+vdecaymode 2      !(1:leptonic decay, 2:muonic decay, 3: tauonic decay,...)
+
+numevts NEVENTS    ! number of events to be generated
+ih1   1           ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1          ! hadron 2 (1 for protons, -1 for antiprotons)
+ndns1 131         ! pdf set for hadron 1 (mlm numbering)
+ndns2 131         ! pdf set for hadron 2 (mlm numbering)
```

Comment:
line  7 and 8 could be removed since external pdfs are used

Reply:
ok, I removed them